src/sign-module.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+echo "Running the Axcient Kernel Module signer"
+
+result=$(mokutil --sb-state)
+if [[ "$result" != *"enabled"* ]]; then
+    echo "Detected a BIOS firmware or the Secure Boot feature is disabled"
+    exit 1
+else 
+    keys=$(cat /proc/keys | grep "asymmetri Axcient")
+    if [ -n "$keys" ]; then 
+	echo "Starting the setup"
+    else
+	echo "WARNING: Secure Boot detected on this system. Either disable EFI Secure Boot "
+	"or check with Axcient knowledgebase for instructions "
+	"for enrolling a signing key for secure DKMS module support." >&2 \
+	&& exit 0
+    fi	
+fi
+
+echo "Kernel Version: ${1}"
+echo "Module Path: ${2}"
+
+hash_algo=sha256
+private_key=/root/axcient-driver.key
+x509_cert=/root/axcient-driver.der
+
+if [  -n "$(uname -a | grep Ubuntu)" ]; then
```

Comment:
Is the path only differ for Ubuntu?
Here is the list of supported configurations.
https://confluence.slc.efscloud.net/pages/viewpage.action?spaceKey=RA&title=Supported+Linux+system+configurations

Reply:
It is (due to the fact that I have ubuntu VM installed). I'll look into these supported distros and see if they need the change there or not.

Reply:
Should be resolved by Konstantin's suggestion. Keeping the conversation up in case of other comments.